src/users/controller/userController.ts

```diff
@@ -39,6 +40,7 @@ export class UsersController {
   @UseInterceptors(ClassSerializerInterceptor)
   @UseGuards(AuthGuard)
   @ApiBearerAuth()
+  @ApiOkResponse({ type: CreateUserDto })
```

Comment:
I see that findAll and find return the same response, then why do we have 2 endpoints?